Addition of Challenge 29 (#697)
OWASP/wrongsecrets: src/main/resources/explanations/challenge29.adoc

=== Secret in a screenshot of a Github issue

A user unknowingly files an issue with a `screenshot` of an actual secret while reporting a bug then realizes it and closes that issue.

Can you spot the secret we are looking for in our GitHub?

> You can easily spot the secret just by looking for it in issues section of our https://github.com/OWASP/wrongsecrets/issues[GitHub].

Reviewer: This does not look like the scenario in the issue, could you rewrite it?

Reviewer 2: Ah no: this is the second scenario (screenshot). First was challenge28

> === Secret in a screenshot of a Github issue
> 
> A user unknowingly files an issue with a `screenshot` of an actual secret while reporting a bug then realizes it and closes that issue.
> 
> Can you spot the secret we are looking for in our GitHub?
> 
> You can easily spot the secret just by looking for it in issues section of our https://github.com/OWASP/wrongsecrets/issues[GitHub].

Reviewer: ```suggestion
=== Secret in a screenshot of a GitHub issue

A user unknowingly files an issue with a `screenshot` of a secret while reporting a bug, then realizes it and closes that issue.

Can you spot the secret we seek in our https://github.com/OWASP/wrongsecrets/issues[GitHub issues]?
```
